frontend/src/components/housing/submission/SubmissionIntakeForm.vue

```diff
@@ -99,7 +100,14 @@ const orgBookOptions: Ref<Array<any>> = ref([]);
 const parcelAccordionIndex: Ref<number | undefined> = ref(undefined);
 const spacialAccordionIndex: Ref<number | undefined> = ref(undefined);
 const validationErrors: Ref<string[]> = ref([]);
-const formUpdated: Ref<boolean> = ref(false);
+const hasFormBeenEdited: Ref<boolean> = ref(false);
```

Comment:
Style: Consider using a more active tense for variable naming. Perhaps `formEdited` or `formModified` could work?